dotnet-desktop-guide/framework/winforms/controls/how-to-group-items-in-a-windows-forms-listview-control.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Group Items in ListView Control
+description: Learn how to group items in LIstView Control, which helps it easier to navigate large lists by grouping items alphabetically, by date or other groupings. 
```

Comment:
Cut "easier."  It reads a little awkwardly.

Reply:
I have replaced the word "help" with "makes".